dubbo-rpc/dubbo-rpc-thrift/src/main/java/org/apache/dubbo/rpc/protocol/thrift/ThriftCodec.java

```diff
@@ -496,6 +499,8 @@ private void encodeRequest(Channel channel, ChannelBuffer buffer, Request reques
             protocol.writeByte(VERSION);
             // service name
             protocol.writeString(serviceName);
+            // path
+            protocol.writeString(inv.getAttachment(Constants.PATH_KEY));
```

Comment:
It seems that we should adjust ThriftCodecTest to make the UT pass.

Reply:
Will do.

Reply:
done